src/azul/indexer/document.py

```diff
@@ -665,6 +685,10 @@ def field_types(cls, field_types: FieldTypes) -> FieldTypes:
         return {
             **super().field_types(field_types),
             'num_contributions': pass_thru_int,
+            'source': {
```

Comment:
So we should either call this `sources` and nest or use the same format as in contributions (two top-level fields). I think the former is more elegant but then we need to also handle the multiplicity in this PR. I don't want to riddle the source with one() calls that would have to be removed later. If that's to difficult, we should do the latter.

Reply:
I'd like to discuss this in parking lot.